applications/ShapeOptimizationApplication/python_scripts/mapper_factory.py

```diff
@@ -16,36 +16,32 @@
 from KratosMultiphysics.ShapeOptimizationApplication import *
 
 # ==============================================================================
-def CreateMapper(model_part, mapper_settings):
+def CreateMapper(origin_model_part, destination_model_part, mapper_settings):
     default_settings = Parameters("""
     {
-        "filter_function_type"       : "linear",
-        "filter_radius"              : 1.0,
-        "max_nodes_in_filter_radius" : 10000,
-        "matrix_free_filtering"      : false,
-        "integration":
-        {
-            "integration_method": "node_sum",
-            "number_of_gauss_points": 0
-        },
-        "consistent_mapping_to_geometry_space": false
+        "filter_function_type"        : "linear",
+        "filter_radius"               : 0.000000000001,
+        "max_nodes_in_filter_radius"  : 10000,
+        "apply_matrix_free_filtering" : false,
+        "apply_consistent_mapping"    : false,
+        "apply_improved_integration"  : false,
+        "integration_method"          : "gauss_integration",
+        "number_of_gauss_points"      : 5
```

Comment:
its better than before, but still not ideal. Because we do not get warned if we have more settings than necessary in the json.
I think the ideal way would be to create one type (`vertex_morphing, vertex_morphing_improved_integration, vertex_morphing_matrix_free` for each cpp class and do the validation of the settings down in cpp.
So we don't have to distribute the logic all over the place in the code.